testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/account/SessionRestServiceTest.java

```diff
@@ -168,14 +173,17 @@ public void testGetDevicesResponse() throws Exception {
         List<ClientRepresentation> clients = session.getClients();
         assertEquals(2, clients.size());
         assertThat(session.getClients(), Matchers.hasItem(Matchers.hasProperty("clientId",
-                Matchers.anyOf(Matchers.is("public-client-0"), Matchers.is("public-client-1")))));
+                anyOf(Matchers.is("public-client-0"), Matchers.is("public-client-1")))));
         assertThat(session.getClients(), Matchers.hasItem(Matchers.hasProperty("clientName",
-                Matchers.anyOf(Matchers.is("Public Client 0"), Matchers.is("Public Client 1")))));
+                anyOf(Matchers.is("Public Client 0"), Matchers.is("Public Client 1")))));
     }
 
     @Test
     public void testGetDevicesSessions() throws Exception {
         ContainerAssume.assumeAuthServerUndertow();
+        assumeTrue("Browser must be htmlunit. Otherwise we are not able to set desired BrowserHeaders",
+                System.getProperty("browser").equals("htmlUnit"));
```

Comment:
A more robust way would probably be something like `driver instanceof HtmlUnitDriver`. But since this is really just a nit pick and it's already merged in -prod repo, let's leave it this way to keep it consistent. ;)